storagenode/console/consoleassets/.gitignore

```diff
@@ -0,0 +1,2 @@
+bindata.go
+initbindata.go
```

Comment:
We already have global ignores for  `*.resource.go`, https://github.com/storj/storj/blob/master/.gitignore#L36

Reply:
initbindata.go isn't really a resource. do you feel strongly about this?

Reply:
You can append the init thing to the resource. Alternatively when you use `cmd/internal/asset` it already creates the init.

Reply:
okay